amr-wind/turbulence/LES/OneEqKsgs.cpp

```diff
@@ -0,0 +1,219 @@
+#include "amr-wind/turbulence/LES/OneEqKsgs.H"
+#include "amr-wind/equation_systems/PDEBase.H"
+#include "amr-wind/turbulence/TurbModelDefs.H"
+#include "amr-wind/derive/derive_K.H"
+#include "amr-wind/turbulence/turb_utils.H"
+#include "amr-wind/equation_systems/tke/TKE.H"
+
+#include "AMReX_ParmParse.H"
+
+namespace amr_wind {
+namespace turbulence {
+
+template <typename Transport>
+OneEqKsgs<Transport>::OneEqKsgs(CFDSim& sim)
+    : TurbModelBase<Transport>(sim), m_vel(sim.repo().get_field("velocity")),
+      m_turb_lscale(sim.repo().declare_field("turb_lscale",1, 1, 1)),
+      m_shear_prod(sim.repo().declare_field("shear_prod",1, 1, 1)),
+      m_buoy_prod(sim.repo().declare_field("buoy_prod",1, 1, 1)),
+      m_rho(sim.repo().get_field("density"))
+{
+    auto& tke_eqn = sim.pde_manager().register_transport_pde(pde::TKE::pde_name());
+    m_tke = &(tke_eqn.fields().field);
+
+    //Turbulent length scale field
+    this->m_sim.io_manager().register_io_var("turb_lscale");
+    this->m_sim.io_manager().register_io_var("shear_prod");
+    this->m_sim.io_manager().register_io_var("buoy_prod");
```

Comment:
use `register_output_var` instead. `io` indicates this will be saved into checkpoint files also and I don't think that is necessary. For output, I feel this should be optional and activated by the user using 

```
io.outputs = turb_lscale shear_prod buoy_prod
```